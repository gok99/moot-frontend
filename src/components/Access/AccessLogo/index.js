import React from 'react';
import { Col } from 'react-bootstrap';

import '../access.css';
import logo from '../../../assets/logo.png';

/**
 * Component for displaying the moot logo on the Access Pages.
 * 
 * @author [Gokul Rajiv] (https://github.com/gok99)
 * @author [Lee Hyung Woon] (https://github.com/lhw-1)
 */
const Access = () => {
  return (
    <Col md="auto">
      <img className="logo-access" src={logo} alt="Moot" />
    </Col>
  );
};

export default Access;