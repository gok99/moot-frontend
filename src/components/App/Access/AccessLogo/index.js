import React from 'react';
import { Col } from 'react-bootstrap';

import '../access.css';
import logo from '../../../../assets/logo.png';

const Access = () => {
  return (
    <Col className="b-accessform d-flex justify-content-md-start">
      <img className="logo-access" src={logo} alt="Moot" />
    </Col>
  );
};

export default Access;