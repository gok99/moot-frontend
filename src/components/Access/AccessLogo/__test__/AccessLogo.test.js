import React from 'react';
import ReactDOM from 'react-dom';
import AccessLogo from '../index';

import { cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect"
import renderer from 'react-test-renderer';

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement('div');
  ReactDOM.render(<AccessLogo />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("matches snapshot", () => {
  const tree = renderer.create(<AccessLogo />).toJSON();
  expect(tree).toMatchSnapshot();
});
