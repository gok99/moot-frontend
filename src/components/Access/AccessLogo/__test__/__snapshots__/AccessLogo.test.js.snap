// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="col-md-auto"
>
  <img
    alt="Moot"
    className="logo-access"
    src="logo.png"
  />
</div>
`;
